src/components/Person/Person.jsx

```diff
@@ -1 +1,40 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable react/jsx-one-expression-per-line */
+export const Person = ({ person }) => {
```

Comment:
think you can use destructing props to get values shorter and more readable e.g.:

```
  const {
    name,
    age,
    partnerName,
    isMarried,
    sex,
  } = person;
```

Reply:
If I use destructuring  props I must to change my code in APP.jsx and then tests do not pass